src/test/java/grumblebug/TaskListTest.java

```diff
@@ -0,0 +1,26 @@
+package grumblebug;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TaskListTest {
+    @Test
+    public void addTest() {
```

Comment:
Could the test functions be named in the three part format `featureUnderTest_testScenario_expectedBehavior()`?